ui/modal_templates.go

```diff
@@ -46,14 +51,37 @@ func (win *Window) LoadModalTemplates() *ModalTemplate {
 	icon.Color = win.theme.Color.Gray
 	icon.Background = win.theme.Color.Surface
 
-	return &ModalTemplate{
+	tpl := &ModalTemplate{
 		th:                    win.theme,
 		confirm:               win.theme.Button(new(widget.Clickable), "Confirm"),
 		cancel:                win.theme.Button(new(widget.Clickable), "Cancel"),
 		walletName:            win.theme.Editor(new(widget.Editor), ""),
 		spendingPassword:      win.theme.Editor(new(widget.Editor), "Spending password"),
 		matchSpendingPassword: win.theme.Editor(new(widget.Editor), "Confirm spending password"),
 		alert:                 icon,
+		moreItemTexts:         []string{"Sign message", "Verify message", "View property", "Privacy", "Rename", "Settings"},
+	}
+	tpl.buttons = make([]*widget.Clickable, len(tpl.moreItemTexts))
+	for i := range tpl.buttons {
+		tpl.buttons[i] = new(widget.Clickable)
+	}
+	return tpl
```

Comment:
This implementation adds complications to the modal_templates. The goal of the templates in modal_templates.go is to create templates that have similar widgets and functions that are re-usable in the same place. You can instead add an option to the global modal implementation to accept custom modal templates. The custom template can then be added as part of the data passed through the channel